csvcubed/csvcubed/readers/vnum.py

```diff
@@ -2,4 +2,4 @@
 
 
 class JsonSchemaVersion(Enum):
-    v1_0 = "V1_0"
\ No newline at end of file
+    v1_0 = "V1_0"
```

Comment:
TODO: Need to replace references to this with the stuff in `schema_versions.py`.